woocommerce-gutenberg-products-block.php

```diff
@@ -41,18 +43,21 @@ function wgpb_initialize() {
  */
 function wgpb_plugins_notice() {
 	echo '<div class="error"><p>';
-	echo __( 'WooCommerce Product Blocks development mode requires files to be built. From the plugin directory, run <code>npm install</code> to install dependencies, <code>npm run build</code> to build the files or <code>npm start</code> to build the files and watch for changes.', 'woo-gutenberg-products-block' );
+	esc_html_e( 'WooCommerce Product Blocks development mode requires files to be built. From the plugin directory, run <code>npm install</code> to install dependencies, <code>npm run build</code> to build the files or <code>npm start</code> to build the files and watch for changes.', 'woo-gutenberg-products-block' );
```

Comment:
There is HTML tags inside this string, probably better change this string or use something that allows `<code>`